galaxy/values-cvmfs.yaml

```diff
@@ -195,12 +195,12 @@ configs:
       database_connection: 'postgresql://{{.Values.postgresql.galaxyDatabaseUser}}:{{.Values.postgresql.galaxyDatabasePassword}}@{{ template "galaxy-postgresql.fullname" . }}/galaxy'
       integrated_tool_panel_config: "/galaxy/server/config/mutable/integrated_tool_panel.xml"
       sanitize_whitelist_file: "/galaxy/server/config/mutable/sanitize_whitelist.txt"
-      tool_config_file: "/galaxy/server/config/tool_conf.xml,{{ .Values.cvmfs.main.mountPath }}/config/shed_tool_conf.xml"
+      tool_config_file: "/galaxy/server/config/tool_conf.xml,/cvmfs/zipped/cvmfs/main.galaxyproject.org/config/shed_tool_conf.xml"
       shed_tool_config_file: "{{.Values.persistence.mountPath}}/config/editable_shed_tool_conf.xml"
-      tool_data_table_config_path: "{{ .Values.cvmfs.main.mountPath }}/config/shed_tool_data_table_conf.xml,{{.Values.cvmfs.data.mountPath}}/managed/location/tool_data_table_conf.xml,{{.Values.cvmfs.data.mountPath}}/byhand/location/tool_data_table_conf.xml"
+      tool_data_table_config_path: "/cvmfs/zipped/cvmfs/main.galaxyproject.org/config/shed_tool_data_table_conf.xml,{{.Values.cvmfs.data.mountPath}}/managed/location/tool_data_table_conf.xml,{{.Values.cvmfs.data.mountPath}}/byhand/location/tool_data_table_conf.xml"
       tool_dependency_dir: "{{.Values.persistence.mountPath}}/deps"
       builds_file_path: "{{.Values.cvmfs.data.mountPath}}/managed/location/builds.txt"
-      datatypes_config_file: "{{ .Values.cvmfs.main.mountPath }}/config/datatypes_conf.xml"
+      datatypes_config_file: "/cvmfs/zipped/cvmfs/main.galaxyproject.org/config/datatypes_conf.xml"
```

Comment:
Are there any tools that use libgalaxy and read the datatypes_config_file? If so, the tool would not have the zipped directory?

Reply:
If you by "use libgalaxy" you mean not have a mulled container and just use the minimal image fallback, yes, we still need to mount the zipped volume into job pods